core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1078,6 +1078,13 @@ package object config {
     .booleanConf
     .createWithDefault(false)
 
+  private[spark] val IGNORE_CORRUPT_FILES_AFTER_RETIES =
```

Comment:
Do we really need another config?
I'm sorta confused, isn't the file effectively 'corrupt' in this case, and you have asked to ignore that?

Reply:
> isn't the file effectively 'corrupt' in this case.

it isn't. an IOE could be a temporary failure.